src/server/lib/middleware/errorHandler.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from 'express';
+
+export const routeErrorHandler = (
+  // eslint-disable-next-line
+  err: any, // ErrorRequestHandler uses type any
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (err.code === 'EBADCSRFTOKEN') {
+    // handle CSRF token errors here
+    res.status(403);
+    res.send('csrf token error');
```

Comment:
we shouldn't just send back a 'csrf token error', need to think about a better way of handling this and displaying it on a page, possibly through the global error somehow 🤔

lets discuss next week!

Reply:
I was thinking for genuine users, with the current implementation this error should never propagate.

Although it could get returned if a user was in the middle of a form when we release the change 🤣